pkgs/applications/audio/neothesia/default.nix

```
@@ -0,0 +1,93 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, cli ? false
+, pkg-config
+, cmake
+, glib
+, alsa-lib
+, atkmm
+, gtk3
+, wrapGAppsHook
+, ffmpeg-full
+, vulkan-loader
+, makeWrapper
+, fontconfig
+, noto-fonts
+, libclang
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "neothesia" + (if cli then "-cli" else "");
```

```suggestion
  pname = "neothesia" + (lib.optionalString cli "-cli");
```

We have a helper function for this